src/bentoml_cli/models.py

```diff
@@ -233,9 +253,37 @@ def import_from(model_path: str) -> None:  # type: ignore (not accessed)
     @click.option(
         "--context", type=click.STRING, default=None, help="Yatai context name."
     )
-    def pull(model_tag: str, force: bool, context: str):  # type: ignore (not accessed)
-        """Pull Model from a yatai server."""
-        cloud_client.pull_model(model_tag, force=force, context=context)
+    @click.option(
+        "-f",
+        "--bentofile",
+        type=click.STRING,
+        default=DEFAULT_BENTO_BUILD_FILE,
+        help="Path to bentofile. Default to 'bentofile.yaml'",
+    )
+    def pull(model_tag: str | None, force: bool, context: str, bentofile: str):  # type: ignore (not accessed)
+        """Pull Model from a yatai server. If model_tag is not provided,
+        it will pull models defined in bentofile.yaml.
+        """
+        if model_tag is not None:
+            cloud_client.pull_model(model_tag, force=force, context=context)
+            return
+
```

Comment:
should we assert that user not providing both model_tag and bentofile (by using `-f bentofile.yaml`? or warn them if they provide both, we will just download model_tag